### ISSUE TITLE
Cannot interact with Avast AV

### ISSUE THREAD

**Issue by vitko-bg** [Ylianst/MeshCentral#1809]
Running on the last MC on Debian with NGINX reverse proxy.
Agent is installed on Win10 x64 and added to Exceptions on Avast AV.
![image](https://user-images.githubusercontent.com/59362521/92929372-58009300-f438-11ea-9e19-4a926a25ad77.png)

Whent trying to disable Avast Shields from the Tray by right clicking:
![image](https://user-images.githubusercontent.com/59362521/92929565-a6ae2d00-f438-11ea-914e-4301e1bc2c7b.png)

The whole screen is freezing... - after 60 sec it restores - when Avast screen disappears(not visible via MC) - connected via TeamViewer to see what is going on.
![image](https://user-images.githubusercontent.com/59362521/92929844-1d4b2a80-f439-11ea-9d7a-89346c67e177.png)

In short - Mesh Agent cannot interact with Avast smart screen by some reason......

**Comment by Ylianst** [comment]
This is a bit of a problem because I suspect these software are purposefully trying to get around having other software disable them. This said, longer term Bryan may work on an improved way of capturing the remote display on Windows to boost speed and the new technique may fix this. This is not something we can fix quickly.

**Comment by vitko-bg** [comment]
It is strange as TeamViewer and AnyDesk can interact, disable and click anywhere on the same client PC.

**Comment by Ylianst** [comment]
At it's root, this problem is the same as #1810.

**Comment by dinger1986** [comment]
@si458 can close

**Comment by si458** [comment]
CLOSED DUPLICATE https://github.com/Ylianst/MeshCentral/issues/1810.
